common/domain/handler_test.go

```diff
@@ -1750,7 +1782,7 @@ func TestHandler_UpdateDomain(t *testing.T) {
 						domainResponse.Info,
 						domainResponse.Config,
 						domainResponse.ReplicationConfig,
-						domainResponse.ConfigVersion,
+						domainResponse.ConfigVersion+1,
```

Comment:
intended behaviour change: Editing cluster attributes causes the ConfigVersion to update